labs/shipped-application-level-policy/3.md

```diff
@@ -0,0 +1,53 @@
+## Create Environment
+
+Click on <b>New Environment</b>, Enter Name of Env "Test" and Save it.
+Our <b>test</b> environment has been created.
+
+![](assets/4.PNG)
+
+
+## Deploy Services 
+
+### Deploy Data Service (Postgres)
+
+Select Successfull build for aie-burnit service from left hand side and, choose the env, that we had created "test". then click deploy. Your app started running.
+
+![](assets/5.PNG)
+
+Click on Env Settings and click on <b>Alerting Thresholds</b>
+
+![](assets/6.PNG)
+
+Scroll Down, Click on Add new Policy
+
+![](assets/7.PNG)
+
+![](assets/8.PNG)
+
+![](assets/9.png)
+
+Click add policy, You policy get added to the service.
+
+Policy Explanation : 
+
+![](assets/10.PNG)
+
+We have defined rule, which signifies that if Memory Use is greater then 75% then Scale Up the container from 1 to 2, and again, if again memory use is greater than 75% with in 2 container it will Scale up by 1 from 2 to 3 container. As we defined 3 as maximum limit so from here it will not scale up.
```

Comment:
We have snapshot of both 51% and 75%.  We should decide values and take snapshot with same value.